test/servinge2e/kourier/servicemesh_test.go

```diff
@@ -43,8 +43,11 @@ type testCase struct {
 
 const (
 	serviceMeshTestNamespaceName = "serverless-tests-mesh"
-	httpProxyImage               = "registry.ci.openshift.org/openshift/knative-v0.17.3:knative-serving-test-httpproxy"
+	httpProxyImage               = "quay.io/openshift-knative-serving-test/httpproxy:v1.3"
+        helloworldImage              = "quay.io/openshift-knative-serving-test/helloworld:v1.3"
 	istioInjectKey               = "sidecar.istio.io/inject"
+        expectedResponse             = "Hello World!"
```

Comment:
This variable seems unused. Please remove it.

Reply:
Resolved